components/NavBar.jsx

```diff
@@ -1,14 +1,30 @@
 import Link from 'next/link';
-import Image from 'next/image';
-import { useRouter } from 'next/router';
 import { Fragment } from 'react';
 import { useSession, signOut } from 'next-auth/react';
 import { Menu, Transition } from '@headlessui/react';
+import { useEffect, useState } from 'react';
 
-export default function NavBar() {
+export default function NavBar({ pathname }) {
```

Comment:
Use `pathname` prop because this prop can be determined at build time. `useRouter` is a React hook and only runs client-side. Not using `useRouter` also lowers bundle size